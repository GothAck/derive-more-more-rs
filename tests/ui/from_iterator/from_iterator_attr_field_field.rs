use derive_more_more::FromIterator;

#[derive(Default, FromIterator)]
struct NamedMultiAttrStructField {
    #[from_iterator(usize)]
    inner: Vec<usize>,
    #[from_iterator(usize)]
    test: bool,
}

#[derive(Default, FromIterator)]
struct UnnamedMultiAttrStructField(#[from_iterator(usize)] Vec<usize>, #[from_iterator(usize)] bool);

fn main() {}
