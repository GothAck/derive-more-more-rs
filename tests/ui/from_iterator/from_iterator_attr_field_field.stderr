error: More than one field with `from_iterator` attribute
 --> tests/ui/from_iterator/from_iterator_attr_field_field.rs:7:5
  |
7 |     #[from_iterator(usize)]
  |     ^

error: More than one field with `from_iterator` attribute
 --> tests/ui/from_iterator/from_iterator_attr_field_field.rs:5:5
  |
5 |     #[from_iterator(usize)]
  |     ^

error: More than one field with `from_iterator` attribute
  --> tests/ui/from_iterator/from_iterator_attr_field_field.rs:12:72
   |
12 | struct UnnamedMultiAttrStructField(#[from_iterator(usize)] Vec<usize>, #[from_iterator(usize)] bool);
   |                                                                        ^

error: More than one field with `from_iterator` attribute
  --> tests/ui/from_iterator/from_iterator_attr_field_field.rs:12:36
   |
12 | struct UnnamedMultiAttrStructField(#[from_iterator(usize)] Vec<usize>, #[from_iterator(usize)] bool);
   |                                    ^
